screenpipe-app-tauri/app/page.tsx

```diff
@@ -53,9 +53,15 @@ export default function Home() {
         toggle_recording_shortcut: settings.recordingShortcut,
```

Comment:
does it save now? let me know

Reply:
Yes it does. I'll make a demo video in an hour or so.